appcelerator-archive/amp
# stack create command, minor refactor

closes https://github.com/appcelerator/amp/issues/361

to test:

`amp stack create -f api/rpc/stack/test_samples/sample-01.yml test`
`curl localhost:90/ping` (error expected)
`amp stack ls` "No stack is available"
`amp stack ls -a` State: Stopped
`amp stack start test`
`sleep 10`
`curl localhost:90/ping` "pong"
